packages/hint/src/lib/config.ts

```diff
@@ -253,9 +251,49 @@ export class Configuration {
         return browserslist(config.browserslist);
     }
 
+    private static groupHintConfigs(configs: ConfigFiles, file: string): ConfigFiles {
+        const basename = path.basename(file);
+
+        if (!basename) {
+            throw new Error(`unrecognised config file '${basename}'`);
+        }
+
+        const hintConfig = Configuration.loadConfigFile(file);
+
+        if (basename === 'package.json') {
+
+            const packageJson = loadJSONFile(file);
+
+            if (packageJson.browserslist) {
+                configs[basename] = packageJson.browserslist;
+            }
+
+            if (hintConfig && hintConfig.browserslist) {
+                configs.hintConfig = hintConfig.browserslist;
+            }
+
+            return configs;
+        }
+
+        if (hintConfig && hintConfig.browserslist) {
+            configs[basename] = hintConfig.browserslist;
+        }
+
+        return configs;
+    }
+
+    private static browserslistConfigError (configKeys: string[]): void {
+
+        const fileNamesJoined = configKeys.length > 2 ?
+            `${configKeys.slice(0, configKeys.length - 1).join(', ')} and ${configKeys[configKeys.length - 1]}`:
+            configKeys.join(' and ');
+
```

Comment:
If we assume this is receiving more than one configKey, I think you don't need the ternary operator.